src/styles/select.scss

```diff
@@ -107,7 +111,7 @@ $block: #{$fd-namespace}-select;
       .#{$block}__text-content {
         @include fd-set-padding-left($fd-select-padding-x-compact);
 
-        min-width: 2rem;
+        min-width: calc(2.75rem - var(--sapField_BorderWidth) * 2);
```

Comment:
repeatable code that can be avoided and easily maintained with a mixin:
```
@mixin fd-select-set-min-width($width) {
     min-width: calc($width - var(--sapField_BorderWidth) * 2);
}
```

and then call it with 2.5rem, 2.75rem and 5rem:
```
@include fd-select-set-min-width(5rem);
```

Reply:
Good catch, thank you!